### DISCUSSION TITLE
Authentication issue

### DISCUSSION THREAD

**Discussion by studentgrant** [srichter/asksonic#27] [Q&A]
Hi there, I'm running Navidrome from a docker container in Unraid - it is publicly accessible via HTTPS. I'm trying to set up asksonic and have followed the instructions to create the skill in the Alexa development area - I can also see it listed under my skills. However, it isn't working - I suspect this is because I haven't provided credentials, as I can't see where I'm supposed to do this. Can anyone assist me here, please?

**Comment by studentgrant** [comment]
I presume I need to set the following variables ASKS_SUBSONIC_USER & ASKS_SUBSONIC_PASS however; I don't see them in the index.js code. Am I looking in the wrong place? Please let me know if I can provide any further details.

**Comment by srichter** [comment]
With docker you'd set those as environment variables

**Reply by studentgrant** [reply]
Hi Srichter, thanks - I'll try setting the three required variables, ASKS_SUBSONIC_URL, ASKS_SUBSONIC_USER, ASKS_SUBSONIC_PASS, on the container and report back.
